docs/getting_started.md

```diff
@@ -64,51 +67,53 @@ EOF
 aws iam create-policy --policy-name kyverno-aws-adapter-policy --policy-document file://my-policy.json
```

Comment:
Good to change `kyverno-aws-adapter-policy` to `<aws-policy-name>`. In our tenant, we have multiple such policies, each adapter user creates their own.

Reply:
Done. Thanks!